package.json

```diff
@@ -48,7 +48,6 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/material": "^5.15.2",
-    "@types/geojson": "^7946.0.14",
```

Comment:
Quickly moved this `@types/` package to the correct spot (`devDependencies`, not regular `dependencies`) while I was here ! Otherwise unrelated to this change.

Reply:
Good catch!